crates/sui-core/src/epoch/randomness.rs

```diff
@@ -47,26 +47,20 @@ pub const SINGLETON_KEY: u64 = 0;
 #[derive(Clone, Debug, PartialEq, Eq, Serialize, Deserialize)]
 #[allow(clippy::large_enum_variant)]
 pub enum VersionedProcessedMessage {
-    V0(dkg_v0::ProcessedMessage<PkG, EncG>),
+    V0(), // deprecated
```

Comment:
again since this is serialized i'm not sure you can change it now?

cc @mystenmark is it ok to change the variant contents like this as long as we know there won't be any actual instances of that variant stored?

Reply:
Assuming we don't try to deser V0, and only V1, it should be safe as the variant indexes remain the same.
This enum is stored in new DB columns, and only with V1 (the V0 messages were stored in the old columns).